Detectors/TRD/calibration/include/TRDCalibration/CalibVDrift.h

```diff
@@ -0,0 +1,68 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_TRD_CALIBVDRIFT_H_
+#define ALICEO2_TRD_CALIBVDRIFT_H_
+
+/// \file   CalibVDrift.h
+/// \author Ole Schmidt, ole.schmidt@cern.ch
+
+#include "DataFormatsTRD/Constants.h"
+#include <array>
+
+namespace o2
+{
+namespace trd
+{
+
+/// \brief VDrift calibration class
```

Comment:
This is future ccdb object, right? Better move it to DataFormats

Reply:
Actually this class is supposed to do the actual calibration (create profile histograms and do some fitting) and create ccdb objects using the results. I am not sure how exactly the ccdb objects will look like, but those can then go into DataFormats.
We wanted to collect all code which does the processing for calibration in TRDCalibration, if that is ok?

Reply:
@martenole OK, I guessed it is an object since did not notice where the input data comes from. So, is this a future `slot` in the time-slot based calibration?

Reply:
Hi Ruben, I am not sure what you mean. This is supposed to get its input from the global tracking. After the matching of TRD tracklets to ITS-TPC tracks a refit is done for the TRD-only part and the difference between the angle of the refits and the tracklets are stored. This is the input for this device. The processing is supposed to be done every ~15 minutes. Is this what you mean with time-slot based calibration?

Reply:
Hi @martenole 
the processing is done in parallel on 250 servers, how are planning to collect the data for 15 min? The standard way is to do the aggregation on separate server, filling the time-slots of detector-defined size, then run the calibration procedure on the content of these slots (or save them for further processing, e.g. the residuals for the SP calibration, yet to be implemented in your workflow). See https://github.com/AliceO2Group/AliceO2/tree/dev/Detectors/Calibration/README.md for details.

Reply:
@shahor02 I see. So yes, this is a future time-slot based calibration device. This PR is supposed to allow for the implementation and testing of the processing methods for calibration. Would it be OK to make the necessary modifications afterwards?
It will need to derive from  `o2::calibration::TimeSlotCalibration<>` and we need to define a class for the input type and for the output container type.